service/controller/app/v1/resource_set.go

```diff
@@ -179,8 +179,6 @@ func NewResourceSet(config ResourceSetConfig) (*controller.ResourceSet, error) {
 	{
 		c := releasemigration.Config{
 			Logger: config.Logger,
-
-			TillerNamespace: config.TillerNamespace,
```

Comment:
Please also delete the flag and from the configmap. I. noticed they were still there while creating https://github.com/giantswarm/app-operator/pull/263.